dbexporter/docker-compose-fem-dbexporter-mongodb.yml

```diff
@@ -7,8 +7,8 @@ services:
   fem_mongodb-exporter:
```

Comment:
Due to consistency, I suggest to rename the container as well. What do you think?